### PR TITLE
T04

### PR DESCRIPTION
**Pull request by dineshN52**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
-    }
+﻿using System;
+using System.Runtime.CompilerServices;
+namespace Training {
+  
+
+   namespace LCMGCD {
+      internal class Program {
+         static int gcd (int a, int b) {
+            while (b != 0) {
```
Comment (ravin-imm) [inline Program.cs:9]: The code is not formatted properly. Why another namespace?

**Review by ravin-imm — CHANGES_REQUESTED** [review]

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+   internal class Program {
+      static int gcd (int a, int b) {
+         while (b != 0) {
+            int temp = b;
+            b = a % b; a = temp;
+         }
+         return a;
+      }
+      static int lcm (int a, int b) {
+         return (a * b) / gcd (a, b);
```
Comment (ravin-imm) [inline Program.cs:13]: Methods/ functions names should start with a capital letter like Gcd (..)

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+         }
+         Console.WriteLine ("Enter the other number");
+         int num2 = int.Parse (Console.ReadLine ());
+         if (num2 <= 0) {
+            Console.WriteLine ("Enter a Positive integer");
+         }
+         int LCM = lcm (num1, num2);
+         int GCD = gcd (num1, num2);
+         Console.WriteLine ($"LCM of {num1} and {num2} is {LCM}");
+         Console.WriteLine ($"GCD of {num1} and {num2} is {GCD}");
```
Comment (ravin-imm) [inline Program.cs:29]: Can be simplified: 
_Console.WriteLine ($"LCM of {num1} and {num2} is {lcm (num1, num2)}")_

File: Program.cs [hunk 4]
```diff
@@ -1,7 +1,33 @@
...
+      }
+      static int lcm (int a, int b) {
+         return (a * b) / gcd (a, b);
+      }
+      static void Main (string[] args) {
+         Console.WriteLine ("Enter the first number");
+         int num1 = int.Parse (Console.ReadLine ());
+         if (num1 <= 0) {
+            Console.WriteLine ("Enter a Positive integer");
+         }
```
Comment (ravin-imm) [inline Program.cs:20]: No need of "{" and "}" brances as its only one line after the _if_ statement.

**Review by ravin-imm — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 4 above)
Comment (dineshN52) [reply Program.cs]: Removed and code updated sir

File: Program.cs (hunk 3 above)
Comment (dineshN52) [reply Program.cs]: Removed the declaration and include the function in WriteLine itself sir

File: Program.cs (hunk 2 above)
Comment (dineshN52) [reply Program.cs]: Changed to capital Letter and updated sir

**Review by ravin-imm — APPROVED** [review]

**Pull request closed without merging**